.circleci/config.yml

```diff
@@ -72,7 +72,10 @@ jobs:
       - run: sudo apt-get install -y expect
       - run: echo 'export NOX_PYTHON_VERSIONS=3.6' >> $BASH_ENV
       - run: sudo pip install nox
-      - run: nox
+      - run:
+          name: "Testing Hydra"
+          no_output_timeout: 60m
```

Comment:
Make is 30 minutes. this is not needed in Linux anyway (There is an issue in circleci where on some operating system there is buffering that prevents output from showing up).
The runs should finish in about 10 minutes currently. 30 minutes is defensive enough.

Reply:
Done